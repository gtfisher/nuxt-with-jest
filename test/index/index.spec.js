import { shallowMount } from '@vue/test-utils'
import index from '../../pages/index'

describe('index', () => {
  test('mounts properly', () => {
    const wrapper = shallowMount(index, {
      stubs: ['Logo', 'NuxtLink'],
    })
    expect(wrapper.vm).toBeTruthy()
  })

  test('renders properly', () => {
    const wrapper = shallowMount(index, {
      stubs: ['Logo', 'NuxtLink'],
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
