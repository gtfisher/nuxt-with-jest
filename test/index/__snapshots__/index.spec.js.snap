// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index renders properly 1`] = `
"<div class=\\"container\\">
  <div>
    <logo-stub></logo-stub>
    <h1 class=\\"title\\">nuxt-with-jest</h1>
    <div class=\\"links\\"><a href=\\"https://nuxtjs.org/\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\" class=\\"button--green\\">
        Documentation
      </a> <a href=\\"https://github.com/nuxt/nuxt.js\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\" class=\\"button--grey\\">
        GitHub
      </a>
      <nuxtlink-stub to=\\"/about\\">About page</nuxtlink-stub>
    </div>
  </div>
</div>"
`;
